### PR TITLE
I18n for EUT4G proposals form

### PR DESCRIPTION
**Pull request by fblupi**
The form is configured with the awesome module

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  eut4g:
+    application_call:
+      fields:
+        name: Nombre
+        email: Email
+        age: Edad
+        gender: Género
+        region: Región/Ciudad
+        country: País
+        applying_as: Está aplicando como...
```
Comment (verarojman) [inline config/locales/es.yml:11]:
```suggestion
        applying_as: Estás aplicando como...
```

File: config/locales/es.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        project_idea: Por favor, describa su idea de proyecto
```
Comment (verarojman) [inline config/locales/es.yml:12]:
```suggestion
        project_idea: Por favor, describe tu idea de proyecto
```

File: config/locales/es.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        actions_to_implement: Por favor, describa las acciones que propone implementar
```
Comment (verarojman) [inline config/locales/es.yml:13]:
```suggestion
        actions_to_implement: Por favor, describe las acciones que propones implementar
```

File: config/locales/en.yml [hunk 4]
```diff
@@ -30,4 +30,35 @@
...
+      options:
+        male: Male
+        female: Female
+        non_binary: Non-binary
+        transgender: Transgender
+        intersex: Intersex
+        undefined: I prefer not to say
+        individual: Individual
+        group_young: Group of young people
+        youth_assoc: Youth association
```
Comment (verarojman) [inline config/locales/en.yml:61]: Could this be renamed to something like `gender_options`? Just wondering, I don't know how custom fields work.

Also, I have a question: Did the client provide this list of options?

**Review by verarojman — CHANGES_REQUESTED** [review]
Hey there! I left some comments, one regarding the naming of the `options` key, which could be more specific (but I don't know if it has to do with the way the "custom fields" feature works)

And others regarding the Spanish form for the verbs (Decidim has the convention of using "tú" instead of "usted" to address participants in Spanish), I did not comment on all of them but you can get the idea :)

Thanks!

File: config/locales/es.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        gender: Género
+        region: Región/Ciudad
+        country: País
+        applying_as: Estás aplicando como...
+        project_idea: Por favor, describe tu idea de proyecto
+        actions_to_implement: Por favor, describe las acciones que propones implementar
+        project_plan: Por favor, define tu plan de proyecto
+        project_relevance: Según tu punto de vista, ¿cuál es la relevancia del proyecto para una transición verde y justa?
+        group_less_exposed_people: ¿Qué grupo objetivo ("personas menos expuestas") has identificado? ¿Puedes describir brevemente por qué consideras que corren el riesgo de ser excluidos de la transición verde?
+        plan_increase_participation: ¿Cómo planeas aumentar la participgsción de este grupo objetivo? ¿Cómo se beneficiarán de tu proyecto?
```
Comment (verarojman) [inline config/locales/es.yml:17]:
```suggestion
        plan_increase_participation: ¿Cómo planeas aumentar la participación de este grupo objetivo? ¿Cómo se beneficiarán de tu proyecto?
```

**Review by verarojman — CHANGES_REQUESTED** [review]
Regarding the gender options, I would rephrase the field name to "Which of the following most accurately describe you?"

And make available options be a list of checkboxes with an "Add your own" option with a free text option, see image below

image

Also, it'd be slighly better if instead of "Male" and "Female" options were "Man" and "Woman", since they are more precise categories in gender terms (Male and Female being more associated with sex than gender)

I know this has nothing to do with code but it has a lot to do with the effort we are trying to make to be more inclusive and provide safe digital spaces 🌈 💜 

Also, these translations can be modified later on with the Term Customizer plugin 😋 

Thanks!

**Sources:**
https://www.ruth-ng.co.uk/how-to-ask-about-gender-in-forms-respectfully
https://denverptc.org/resource.php?id=231
https://uxdesign.cc/designing-forms-for-gender-diversity-and-inclusion-d8194cf1f51

File: config/locales/es.yml (hunk 5 above)
Comment (fblupi) [reply config/locales/es.yml]: Nice! I have changed the form with these recommendations and I have updated the locales too

File: config/locales/es.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+es:
+  eut4g:
+    application_call:
+      fields:
+        name: Nombre
+        email: Email
+        age: Edad
+        gender: ¿Cuál de las siguientes opciones te describen mejor?
```
Comment (verarojman) [inline config/locales/es.yml:8]:
```suggestion
        gender: ¿Cuales de las siguientes opciones te describen mejor?
```

**Review by verarojman — DISMISSED** [review]
Perfect! I'm approving and left just one tiny suggestion :)

Great work 🙌🏾

**Review by verarojman — COMMENTED** [review]
👌🏾
